addons/website/static/tests/tour_utils/widget_lifecycle_patch_wysiwyg.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module **/
+
+import Wysiwyg from "web_editor.wysiwyg";
+import { patch } from "@web/core/utils/patch";
+
+const patchWysiwyg = () => patch(Wysiwyg.prototype, "widget_lifecycle_patch_wysiwyg.wysiwyg", {
+    /**
+     * @override
+     */
+    async start() {
+        window.localStorage.WidgetAndWysiwygLifecycle = JSON.stringify(
+            JSON.parse(window.localStorage.WidgetAndWysiwygLifecycle).concat("wysiwygStart"));
+        await this._super(...arguments);
+        window.localStorage.WidgetAndWysiwygLifecycle = JSON.stringify(
+            JSON.parse(window.localStorage.WidgetAndWysiwygLifecycle).concat("wysiwygStarted"));
+    },
+    /**
+     * @override
+     */
+    destroy() {
+        this._super(...arguments);
+        // Two "destroy" will happen during this test: one when leaving the edit
+        // mode (see "leaveEditMode()" of the wysiwyg_adapter) and another due
+        // to the owl process as the wysiwyg is removed from the DOM. Do not
+        // take the second one into account as it is not relevant in the
+        // order of the lifecycle method call of the wysiwyg and the widget.
```

Comment:
Is not that a bug to fix? Surely it can't be good that the destroy method is called twice?